README.md

```diff
@@ -1,2 +1,27 @@
-# codeuw
-Repository of resources and tasks for contributing to scientific software projects
+# CodeUW
+
+CodeUW gives UW students the opportunity to gain hands-on coding experience by contributing to scientific software. The repo maintains a collection
+of tasks across a variety of projects from the [Scientific Software Engineering Center (SSEC)](https://escience.washington.edu/software-engineering/ssec/). The tasks are defined by SSEC engineers and are meant to be small and self-contained pieces of work that can be done in under a week. 
+
+By working on these tasks, students have the opportunity to work with SSEC engineers, who will act as mentors, review the code and provide feedback.
+
+
+## Pre-requisites
+
+- Being a UW student
+- Familiarity with [GitHub](https://github.com):
+    - Having a github account
+    - Knowledge of basic operations, such as pulling/forking a repo, creating branches, making commits and pushing. 
```

Comment:
Anyone have a good tutorial for basic Git operations we can point to?

Reply:
I like this resource for open source development: https://www.dataschool.io/how-to-contribute-on-github/

Reply:
Thanks! Added a link